src/lib/viewers/controls/color-picker/ColorPickerControl.scss

```diff
@@ -9,6 +15,13 @@
 
 .bp-ColorPickerControl-button {
     @include bp-ControlButton;
+
+    &:hover,
```

Comment:
By moving the `bp-is-active` class to the button, it can be combined with the hover and focus pseudo-classes and the styles within the mixin can be inlined.